test/unit/test-standard-actions.js

```diff
@@ -560,6 +560,32 @@ describes.sandboxed('StandardActions', {}, (env) => {
       standardActions.handleToggleChecked_(invocation);
       expectCheckboxToHaveCheckedStateTrue(element);
     });
+    
```

Comment:
The linting issue is here, there's trailing whitespace on this line.

```suggestion

```

Reply:
> The linting issue is here, there's trailing whitespace on this line.

Thank you very much! It's now fixed at https://github.com/anrghg/amphtml/blob/2471a18d9bc1b0b12f63bc9f87d76c753693832b/test/unit/test-standard-actions.js#L563